spra-play-server/src/main/scala/net/wiringbits/spra/admin/repositories/daos/DAOUtils.scala

```diff
@@ -0,0 +1,13 @@
+package net.wiringbits.spra.admin.repositories.daos
+
+import java.util.UUID
+
+object DAOUtils {
+  def validateIsUUID(uuid: String): Option[UUID] = {
```

Comment:
I'd rename this to `validateUUID` because this returns the UUID instead of a boolean